modules/ROOT/pages/index.adoc

```diff
@@ -63,215 +93,10 @@ xref:runtime-manager::anypoint-platform-cli-commands.adoc[Anypoint CLI 3.x],
-
-Mule supports batch jobs, which split messages into individual records,
-act on each record, then report the results and, if necessary, push the processed
-output to other systems or queues. This functionality is particularly useful when
-working with streaming input or when engineering "near real-time" data integration
-between SaaS applications.
-
-Studio and Design Center provide various batch components that you can use to
-configure batch processes. For more on this topic, see
-xref:batch-processing-concept.adoc[Batch Processors].
-
-== API Gateway
-
-Mule Runtime includes an embedded API Gateway, which enables you to apply
-security policies to an API, enrich incoming or outgoing messages, and add
-capabilities to an API without having to write any code. See
-xref:api-manager::runtime-agw-landing-page.adoc[API Gateway]
-for information on the Mule runtime engine in the API Manager documentation.
-
-== Non-blocking Execution Engine
-
-The Mule execution engine is based on a non-blocking runtime. This is a
-task-oriented execution model that allows you to take advantage of non-blocking
-IO calls and to make tuning configurations in a simple way.
-
-Flows always function synchronously. If you need to achieve asynchronous
-patterns, such as Fire-and-Forget, you can use the xref:async-scope-reference.adoc[Async (`<async>`)] component.
-
-Mule Event processors indicate to the runtime whether they are CPU intensive,
-CPU light, or IO intensive operations. These workload types help the runtime tune
-for different workloads, removing the need for you to manage thread pools manually
-to achieve optimum performance. Instead, the runtime introspects the available
-resources (such as memory and CPU cores) in the system to tune thread pools
-automatically.
-
-For more detailed documentation on the execution engine, threading model, and
-tuning options, see xref:execution-engine.adoc[Execution Engine].
-
-[[version]]
-== Mule Versioning
-
-The Mule version you use determines what your Mule app can do and what
-features and products are compatible with Mule.
-xref:about-components.adoc[Core components], which process the Mule event
-as it travels through flows in a Mule app, are part of a Core module that is
-bundled with and shares the same version as Mule.
-xref:connectors::index.adoc[Modules, connectors], the
-xref:dataweave-language-guide.adoc[DataWeave language], and a number of MuleSoft
-products have their own versioning system but are compatible with specific
-versions of Mule. For example, DataWeave 2.0 and Studio 7.x are compatible
-Mule 4.x runtimes, while DataWeave 1.0 and Studio 6.x are compatible with
-Mule 3.x runtimes. You need to make sure the connector or module you
-use in a Mule app is compatible with your Mule version.
+== See Also
 
-//TODO? LINK TO A COMPATIBILITY MATRIX OR SOMETHING. DOES ONE EXIST? PROB SHOULD.
+* xref:mule-app-dev.adoc[Mule Application Development]
+* xref:studio::domain-support-concept.adoc[Domains]
```

Comment:
Why is this ref. here when the topic is included above?

Reply:
The full topic is described in a different repo & section of the docs. We have to point people out to it.